reference/dockerfile/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Dockerfile reference
+description: "Dockerfiles use a simple DSL which allows you to automate the steps you would normally manually take to create an image."
+keywords: build, dockerfile, reference
+toc_max: 3
+redirect_from:
+- /engine/reference/builder/
+- /reference/builder/
+---
+
+{% include_relative reference.md %}
```

Comment:
@thaJeztah as discussed I have created a stub file and moved the frontmatter section from buildkit repo to the stub file so we have control over it in docs repo.